exercises/sublist/test/test_sublist.c

```diff
@@ -0,0 +1,198 @@
+#include "vendor/unity.h"
+#include "../src/sublist.h"
+
+void test_empty_lists(void)
+{
+   TEST_ASSERT_EQUAL(EQUAL, check_lists(NULL, NULL, 0, 0));
+}
+
+void test_empty_list_within_non_empty_list(void)
+{
+   int listTwo[] = { 1, 2, 3 };
+
+   TEST_ASSERT_EQUAL(SUBLIST, check_lists(NULL, listTwo, 0, sizeof(listTwo)));
```

Comment:
Passing the length argument as the size in bytes is pretty weird, and I don't think we do it in other array exercises. The length argument should instead be the number of elements.

Reply:
Yeah, I initially had it passing the number of elements and then I somehow convinced myself that it was weird, so I changed it.  But, now I agree that it was right the first time.